game/core/src/main/java/com/gg/game/Model/World.java

```diff
@@ -99,4 +106,6 @@ public void setBackground(TextureType texture){
     public Vector getPlayerPos() {
         return player.position;
     }
+
+    public Vector getEnemiesPos(){return levels.get(0).getCurrentRoom().getEnemies().get(0).getPos();}
```

Comment:
It knows too much. Change.

```suggestion
    public Vector getEnemiesPos(){return levels.get(0).getEnemyPosition();}
```

Reply:
Neither current or suggested change works once Room contains multiple Enemy. It's a quick fix that will be resolved. 
```suggestion
    public Vector[] getEnemiesPos(){return currentLevel.getEnemisPositions();}
```
Or something along that style